### PR TITLE
Javelin Bag

### PR DESCRIPTION
**Pull request by EirenexTheDragon**
A Bag to store Javelins + Crafting + Flags for the 2 Vanilla Javelins.
More Survival tool has stone javelin, whoever is managing that, can put the Flag [JAVELIN] into the flag of it to allow it to store that javelin in the bag too, if thats possible at all since its a mod kinda.

### CODE REVIEW

**Comment by Maddremor** [comment]
I don't really get how or why you would store javelins in a bag. Like a really big quiver?

**Comment by EirenexTheDragon** [comment]
the sense for storing javelin is, that one javelin is like 1 volumen, if you have a proper quiver to store them, it allows you to free your space in your inventory for looting or whatever else. here is a picture of an javelin bag 
grafik
its from a game, but they have been used in medieval times. there also are ancient ways of carrying spears, like the antic times, where skirmishers usually would hold 2-3 spears in their of hand to throw extra, before retreating.

File: data/json/items/armor.json [hunk 1]
```diff
@@ -9609,5 +9609,32 @@
       "draw_cost": 40,
       "flags": [ "SHEATH_GOLF" ]
     }
-  }
+  },
+  {"id": "javelin_bag",
```
Comment (Night-Pryanik) [inline data/json/items/armor.json]: `id` field should be on the next line after the first.

File: data/json/items/armor.json [hunk 2]
```diff
@@ -9609,5 +9609,32 @@
...
+    "type": "ARMOR",
+    "name": "javelin bag",
+    "name_plural": "javelin bags",
+    "description": "A open medieval looking bag, designed to store javelins for easy reach.",
```
Comment (Night-Pryanik) [inline data/json/items/armor.json]: An open. Medieval-looking.

File: data/json/items/armor.json [hunk 3]
```diff
@@ -9609,5 +9609,32 @@
...
+    "weight": 600,
+    "volume": 5,
+    "price": 6000,
+    "rigid": false,
+    "material": "leather",
+    "symbol": "[",
+    "color": "brown",
+    "covers": [ "TORSO" ],
+    "coverage": 30,
+	 "encumbrance": 10,
```
Comment (Night-Pryanik) [inline data/json/items/armor.json]: Incorrect indentation.

File: data/json/recipes/armor/storage.json [hunk 4]
```diff
@@ -797,5 +797,18 @@
...
       [ [ "leather_pouch", 5 ] ]
     ]
+  },
+  {
+    "result": "javelin_bag",
+    "type": "recipe",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_STORAGE",
+    "skill_used": "tailor",
+	 "skills_required": [ [ "gun", 1 ], [ "throw", 1 ] ],
```
Comment (Night-Pryanik) [inline data/json/recipes/armor/storage.json]: Incorrect indentation.

**Comment by ZhilkinSerg** [comment]
Still should be linted, but I'll handle it while merging.

**Pull request merged**